src/jdk.naming.dns/share/classes/com/sun/jndi/dns/DnsClient.java

```diff
@@ -158,6 +159,16 @@ public DnsClient(String[] servers, int timeout, int retries)
             ne.setRootCause(e);
             throw ne;
         }
+        // register a cleaning action to close the channel
+        // selector when this DNS client becomes phantom reachable
```

Comment:
The change looks okay, it's just the comment here isn't quite right, it should say "to close the Selector when ...".

Reply:
Thanks Alan. Fixed the comment in 757d242c66cdd5f12ce695a4c80ce5700eb66b4e.